locales-pending/premium.ftl

```diff
@@ -23,6 +23,16 @@ toolbar-app-picker-product-fx-desktop = { -brand-fx-desktop }
 toolbar-app-picker-product-fx-mobile = { -brand-fx-mobile }
 toolbar-app-picker-by-mozilla = Made by { -brand-mozilla }
 
+user-menu-trigger-label = Open menu
```

Comment:
Not sure if this is the best alternative, but it might at least be good to distinguish this from the other menus. Maybe something with "Profile", given the tooltip?

```suggestion
user-menu-trigger-label = Open user menu
```